src/routes/categories/components/CategoryListItem.js

```diff
@@ -27,7 +27,8 @@ class CategoryListItem extends React.Component {
         {children.map(child =>
           <div key={child.id} className={style.subRow}>
             <Link href={child.url}>
-              <img src={child.thumbnail || iconPlaceholder} className={style.categoryThumbnail} />
+              {(child.thumbnail && <img src={child.thumbnail} className={style.categoryThumbnail} />) ||
```

Comment:
imo `{child.thumbnail ? <img src={child.thumbnail} className={style.categoryThumbnail} /> : <div className={style.categoryThumbnail} />` is easier to understand

Reply:
Or even more readable, extract it to a function :)